SConstruct

```diff
@@ -114,6 +114,25 @@ def is_platform_arm():
         return True
     return False
 
+def check_asan_helper(context):
```

Comment:
I can't see what this is doing that's asan specific?

Reply:
I should probably move the options into this routine